{{cookiecutter.repo_name}}/runtime.txt

```diff
@@ -1 +1 @@
-python-3.4.2
+python-3.5.0
```

Comment:
It can be 3.5.2 on heroku and 3.5.0 on circle. They're managed in different config files